pages/team.html

```diff
@@ -29,7 +29,7 @@ <h2> Team Members</h2>
               <div class="card-banner">
                   <div class="team-list-img" role="group">
                     {% if teamMember.image != nil %}
-                      <img src="{{ teamMember.image }}" alt="Photo of {{ teamMember.full_name }}">
+                      <img src="{{ teamMember.image }}" alt="Team member named {{ teamMember.full_name }}">
```

Comment:
I like the "photo of {name}" better. Because it really is a photo of that person.

Reply:
If I'm not wrong as long as there's an img tag, the screen reader will say it's an image. Therefore the alt just needs to describe what it is rather than saying it's an image/picture of